src/pa02/snakes_simulations.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Håvard Brobakken Eig, Olav vikøren Espenes '
+__email__ = 'havardei@nmbu.no, olaves@nmbu.no'
+
+import random
+
+
+class Board:
+    """
+    Defining the board with its snakes, ladders and the goal.
+    """
+    def __init__(self, *args, **kwargs):
```

Comment:
This is cheating. And you cannot customize your ladders / snakes / goal with this implementation.